templates/payment-form.html

```diff
@@ -22,16 +22,57 @@ <h1>
   <div class="column">
     <form class="payment-form" id="fast-checkout">
       <div class="wrapper">
-        <h3>Contact Info</h3>
-        <div id="details-wrapper" class="details-wrapper">
-          <label for="firstname">First name:</label>
-          <input type="text" id="firstname" name="firstname">
-          <label for="lastname">Last name:</label>
-          <input type="text" id="lastname" name="lastname">
-          <label for="email">Email:</label>
-          <input type="email" id="email" name="email">
-          <label for="details">Details:</label>
-          <textarea id="details" name="details">Share your job details here...</textarea>
+        <h3>Job Listing Form</h3>
+        <div id="info-wrapper" class="info-wrapper">
+          <fieldset>
+            <legend>Contact Information</legend>
+
+            <label for="firstname">First Name</label>
+            <input type="text" id="firstname" name="firstname" required>
+
+            <label for="lastname">Last Name</label>
+            <input type="text" id="lastname" name="lastname" required>
+
+            <label for="email">Email</label>
+            <input type="email" id="email" name="email" required>
+          </fieldset>
+
+          <fieldset>
+            <legend>Job Details</legend>
+
+            <label for="jobtitle">Job Title</label>
+            <input type="text" id="jobtitle" name="jobtitle" required>
+
+            <label for="company">Company</label>
+            <input type="text" id="company" name="company" required>
+
+            <label for="type">Type <span>(optional)</span></label>
+            <input type="text" id="type" name="type">
+
+            <label for="educationreq">Education Requirement <span>(optional)</span></label>
+            <input type="text" id="educationreq" name="educationreq">
+
+            <label for="location">Location</label>
+            <input type="text" id="location" name="location" required>
+
+            <div id="radio-wrapper">
+              <p>Can you make a referral/connection?</p>
+              <label for="ref-yes">Yes</label>
+              <input type="radio" id="ref-yes" name="referral" value="yes">
+              <label for="ref-no">No</label>
+              <input type="radio" id="ref-no" name="referral" value="no">
+            </div>
+
+            <label for="salaryrange">Salary Range <span>(optional)</span></label>
+            <input type="text" id="salaryrange" name="salaryrange">
+
+            <label for="description">Description <span>(optional)</span></label>
+            <textarea id="description" name="description">Write job details here...</textarea>
```

Comment:
Change description to `Submit your job for $100 to be shared with our 100+ full-stack software engineer graduates.`

Reply:
Addressed in [2c5745b](https://github.com/Techtonica/techtonica.org/pull/346/commits/2c5745b9a1d121f0d16e42983fed9b20450329c1)